### ISSUE TITLE
A11Y: Cookie details not reachable using keyboard alone

### ISSUE THREAD

**Issue by nlehmkuhl** [eibiflo/cf_cookiemanager#50]
(Frontend)
It is not possible to view the cookie details using the keyboard.
There is only a click-event on the table row which is not focusable.

Possible solution:
Wrap the action info icon in a button and add the click-event too.
